File: Plugins/Aspose-Pdf-Java-for-Jython/asposepdf/WorkingWithPages/GetPageProperties.py
from asposepdf import Settings
from com.aspose.pdf import Document
from java.util import Date


class GetPageProperties:

    def __init__(self):
        
        dataDir = Settings.dataDir + 'WorkingWithPages/GetPageProperties/'

        # Create PDF document
        pdf_document = Document(dataDir + 'input1.pdf');

        # get page collection
        page_collection =pdf_document.getPages();

        # get particular page
        pdf_page =page_collection.get_Item(1);

        #get page properties
        print "ArtBox : Height = " 
        print pdf_page.getArtBox().getHeight()
        print ", Width = " 
        print pdf_page.getArtBox().getWidth()
        print ", LLX = " 
        print pdf_page.getArtBox().getLLX()
        print ", LLY = "
        print pdf_page.getArtBox().getLLY()
        print ", URX = " 
        print pdf_page.getArtBox().getURX() 
        print ", URY = " 
        print pdf_page.getArtBox().getURY()
        
        
        print "BleedBox : Height = " 
        print pdf_page.getBleedBox().getHeight() 
        print ", Width = " 
        print pdf_page.getBleedBox().getWidth()
        print ", LLX = " 
        print pdf_page.getBleedBox().getLLX() 
        print ", LLY = " 
        print pdf_page.getBleedBox().getLLY() 
        print ", URX = " 
        print pdf_page.getBleedBox().getURX()
        print ", URY = " 
        print pdf_page.getBleedBox().getURY()
        
        
        print "CropBox : Height = " 
        print pdf_page.getCropBox().getHeight() 
        print ", Width = " 
        print pdf_page.getCropBox().getWidth() 
        print ", LLX = " 
        print pdf_page.getCropBox().getLLX() 
        print ", LLY = " 
        print pdf_page.getCropBox().getLLY() 
        print ", URX = " 
        print pdf_page.getCropBox().getURX() 
        print ", URY = " 
        print pdf_page.getCropBox().getURY()
        
        
        print "MediaBox : Height = " 
        print pdf_page.getMediaBox().getHeight() 
        print ", Width = " 
        print pdf_page.getMediaBox().getWidth() 
        print ", LLX = "
        print pdf_page.getMediaBox().getLLX() 
        print ", LLY = " 
        print pdf_page.getMediaBox().getLLY() 
        print ", URX = " 
        print pdf_page.getMediaBox().getURX() 
        print ", URY = " 
        print pdf_page.getMediaBox().getURY()
        
        print "TrimBox : Height = " 
        print pdf_page.getTrimBox().getHeight()
        print ", Width = "
        print pdf_page.getTrimBox().getWidth() 
        print ", LLX = " 
        print pdf_page.getTrimBox().getLLX() 
        print ", LLY = " 
        print pdf_page.getTrimBox()
        print getLLY() 
        print ", URX = " 
        print pdf_page.getTrimBox().getURX() 
        print ", URY = " 
        print pdf_page.getTrimBox().getURY()
        
        
        print "Rect : Height = "
        print pdf_page.getRect().getHeight() 
        print ", Width = " 
        print pdf_page.getRect().getWidth()
        print ", LLX = "
        print pdf_page.getRect().getLLX()
        print ", LLY = " 
        print pdf_page.getRect().getLLY()
        print ", URX = " 
        print pdf_page.getRect().getURX()
        print ", URY = " +pdf_page.getRect().getURY()
        
        
        print "Page Number :"
        print pdf_page.getNumber()
        print "Rotate :" 
        print pdf_page.getRotate()
 
if __name__ == '__main__':        
    GetPageProperties()

File: Examples/src/main/java/com/aspose/pdf/examples/AsposePdfExamples/Pages/GetPageProperties.java
package com.aspose.pdf.examples.AsposePdfExamples.Pages;

import com.aspose.pdf.Document;
import com.aspose.pdf.Page;
import com.aspose.pdf.PageCollection;

public class GetPageProperties {

	public static void main(String[] args) {
		// Open a document
		Document pdfDocument = new Document("input.pdf");
		// Get the page collection
		PageCollection pageCollection = pdfDocument.getPages();
		// Get a specific page
		Page pdfPage = pageCollection.get_Item(1);
		// Get the page properties
		System.out.printf("\n ArtBox : Height = " + pdfPage.getArtBox().getHeight() + ", Width = " + pdfPage.getArtBox().getWidth() + ", LLX = " + pdfPage.getArtBox().getLLX() + ", LLY = " + pdfPage.getArtBox().getLLY() + ", URX = " + pdfPage.getArtBox().getURX() + ", URY = " + pdfPage.getArtBox().getURY());
		System.out.printf("\n BleedBox : Height = " + pdfPage.getBleedBox().getHeight() + ", Width = " + pdfPage.getBleedBox().getWidth() + ", LLX = " + pdfPage.getBleedBox().getLLX() + ", LLY = " + pdfPage.getBleedBox().getLLY() + ", URX = " + pdfPage.getBleedBox().getURX() + ", URY = " + pdfPage.getBleedBox().getURY());
		System.out.printf("\n CropBox : Height = " + pdfPage.getCropBox().getHeight() + ", Width = " + pdfPage.getCropBox().getWidth() + ", LLX = " + pdfPage.getCropBox().getLLX() + ", LLY = " + pdfPage.getCropBox().getLLY() + ", URX = " + pdfPage.getCropBox().getURX() + ", URY = " + pdfPage.getCropBox().getURY());
		System.out.printf("\n MediaBox : Height = " + pdfPage.getMediaBox().getHeight() + ", Width = " + pdfPage.getMediaBox().getWidth() + ", LLX = " + pdfPage.getMediaBox().getLLX() + ", LLY = " + pdfPage.getMediaBox().getLLY() + ", URX = " + pdfPage.getMediaBox().getURX() + ", URY = " + pdfPage.getMediaBox().getURY());
		System.out.printf("\n TrimBox : Height = " + pdfPage.getTrimBox().getHeight() + ", Width = " + pdfPage.getTrimBox().getWidth() + ", LLX = " + pdfPage.getTrimBox().getLLX() + ", LLY = " + pdfPage.getTrimBox().getLLY() + ", URX = " + pdfPage.getTrimBox().getURX() + ", URY = " + pdfPage.getTrimBox().getURY());
		System.out.printf("\n Rect : Height = " + pdfPage.getRect().getHeight() + ", Width = " + pdfPage.getRect().getWidth() + ", LLX = " + pdfPage.getRect().getLLX() + ", LLY = " + pdfPage.getRect().getLLY() + ", URX = " + pdfPage.getRect().getURX() + ", URY = " + pdfPage.getRect().getURY());
		System.out.printf("\n Page Number :- " + pdfPage.getNumber());
		System.out.printf("\n Rotate :-" + pdfPage.getRotate());
	}
}
